cscareerhub/megabot
# [feature] Create list command for AMA bot

### Purpose of pull request

**Related Issue**: resolves #49 

Creates List and basic addition (more for sake of testing) of AMAs. List Appearance is below:
![image](https://user-images.githubusercontent.com/1833579/101553525-cf37bb80-3982-11eb-9b34-c873e2beb263.png)


### Pull request checklist


### Testing instructions

Unit tests added.
For manual testing:
- add events doing command: `++ama add yyyy-mm-dd Event Title`
- list events with `++ama list` and events should come up on screen
